internal/app/connectconformance/testsuites/data/connect_client_code_to_http_code.yaml

```diff
@@ -1,12 +1,12 @@
 name: Connect to HTTP Code Mapping
+# These tests verify that an explicit Connect code maps to the correct HTTP code
+# according to the protocol.
 mode: TEST_MODE_CLIENT
 relevantProtocols:
   - PROTOCOL_CONNECT
-# These tests verify that an explicit Connect code maps to the correct HTTP code
-# according to the protocol.
 testCases:
 - request:
-    testName: unary canceled mapping
+    testName: unary/canceled
```

Comment:
Same here: these are testing HTTP code mapping and all of them are unary. In fact, it is only the unary connect protocol that even uses HTTP status codes this way, so the "unary/" prefix is redundant in that regard as well. You might even rename the test suite and file to  "Connect Unary to HTTP Code Mapping".

Reply:
There are actually other tests for all the other stream types in this file too that test a `200` response on stream errors. I think the convention of 'if the file contains multiple stream types, prefix the test name with the stream type' is probably a good one.  WDYT?

Reply:
My bad. When I scanned through the diff, all I saw was "unary". But I now see that, while most are unary, there are a handful of others below. So disregard this comment.